app/views/admin/booths/index.html.haml

```diff
@@ -9,6 +9,22 @@
             = link_to 'Add Booth', new_admin_conference_booth_path(@conference.short_title), class: 'button btn btn-primary'
       %p.text-muted
         All the booth requests
+
+      - if @conference.booth_limit == 0
+        %p
+          Set the
+          = link_to 'Booth limit', edit_admin_conference_path(@conference.short_title)
+          to make sure you are not accepting more booth requests than possible.
+      - elsif !maximum_accepted_booths?
+        %p
+          You cannot accept more than
+          %b
+            = @conference.booth_limit
+          booth requests. (
+          = pluralize(@conference.booths.where(state: 'accepted').count, 'accepted booth')
```

Comment:
This could be moved into a scope so that we can simply do @conference.booths.accepted